functions_and_scope_lab.js

```diff
@@ -0,0 +1,289 @@
+// # Function Exercises
```

Comment:
Good demonstration of how to use and implement functions. No major feedback here.